features/device-mgt/org.wso2.carbon.device.mgt.server.feature/src/main/resources/dbscripts/cdm/postgresql.sql

```diff
@@ -8,8 +8,8 @@ CREATE TABLE IF NOT EXISTS  DM_DEVICE_TYPE (
 CREATE TABLE IF NOT EXISTS DM_DEVICE_CERTIFICATE (
   ID                    BIGSERIAL NOT NULL PRIMARY KEY,
   SERIAL_NUMBER         VARCHAR(500) DEFAULT NULL,
-  CERTIFICATE           BYTEA DEFAULT NULL
-  TENANT_ID INTEGER DEFAULT 0,
+  CERTIFICATE           BYTEA DEFAULT NULL,
+  TENANT_ID INTEGER DEFAULT 0
```

Comment:
Fix formatting.

Reply:
Fixed in f45528147c4c960bf429dd92c68b08a5359239fa